### PR TITLE
Remove 9gag

### LINKED ISSUE #4: Remove 9GAG list item
# Remove 9GAG

We are not going to implement 9GAG page at all since the API is not really open. Please remove it.

* Remove category from `MainViewModel.categories`
* Remove part of the code in `MainView` which opens the "Coming Soon" when user taps 9gag
* Remove `9gag` image from `Assets`
* Remove `ComingSoon.swift` file

### PR DESCRIPTION
**Pull request by qetiavanashvili**
closes https://github.com/ashalva/ios.memenator/issues/4

Remove category from MainViewModel.categories
Remove part of the code in MainView which opens the "Coming Soon" when user taps 9gag
Remove 9gag image from Assets
Remove ComingSoon.swift file

### CODE REVIEW

**Comment by ashalva** [comment]
@qetiavanashvili couple of improvements before I go through the changes:

1. You should assign yourself to Assignees.
2. You should assign me and @georgemakadze as Reviewers.
3. You have to use correct url for your issue -> https://github.com/ashalva/ios.memenator/issues/4

**Comment by qetiavanashvili** [comment]
@ashalva

**Comment by qetiavanashvili** [comment]
@ashalva  Please, check the assignment

File: GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "images" : [
     {
-      "filename" : "9gag.svg",
```
Comment (ashalva) [inline GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: You deleted only the image, but we need to delete entire file. So, please just delete entire file of 9gag in `Assets`

image

**Review by ashalva — COMMENTED** [review]
Looking good, you need to fix only one issue before I approve ✅

File: GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json (hunk 1 above)
Comment (qetiavanashvili) [reply GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: Should  I delete only 9gag file? Does it need some changes in terminal?

Comment (qetiavanashvili) [reply GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: I have deleted the file of 9gag, is it correct now? I haven't done anything else.

Comment (qetiavanashvili) [reply GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: I think to make a git push on terminal, but I am not quite sure, that's why I am waiting to you

Comment (ashalva) [reply GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: You need to delete file in Xcode and thats it. After that just push the changes. I think it should be correct, if not we can easily revert the changes, dont worry.

Comment (qetiavanashvili) [reply GeorgianMemenator/Assets.xcassets/Icons/9gag.imageset/Contents.json:4]: I have done everything, please check

**Review by ashalva — APPROVED** [review]
Super @qetiavanashvili, well done 👏 
Its time to merge your first pull request 🎉

**Comment by qetiavanashvili** [comment]
Thank you. I am very pleased👌

**Pull request merged**